pandas/core/indexing.py

```diff
@@ -449,6 +449,37 @@ def loc(self) -> _LocIndexer:
         8          4       5
         9          7       8
 
+        **Set values with a Series**
+
+        Set column values using a Series
+
+        >>> df.loc[:, 'shield'] = pd.Series({7: 8, 8: 10, 9: 13})
+        >>> df
+           max_speed  shield
+        7          1       8
+        8          4       10
+        9          7       13
+
+        Setting values using a Series with inconsistent indexes aligns the
```

Comment:
Then this can just state where the index values are not found in the Series index, the values will be `NaN`